eng/Version.Details.xml

```diff
@@ -84,6 +84,10 @@
     <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="7.0.100-alpha.1.21427.3" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/templating</Uri>
       <Sha>dee13998a1d3318ab7363aaa7c8b095d3e58e66b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.TemplateEngine.Cli" Version="7.0.100-alpha.1.21427.3" CoherentParentDependency="Microsoft.NET.Sdk">
+      <Uri>https://github.com/dotnet/templating</Uri>
+      <Sha>dee13998a1d3318ab7363aaa7c8b095d3e58e66b</Sha>
       <SourceBuild RepoName="templating" ManagedOnly="true" />
```

Comment:
@dseefeld For this change. Microsoft.DotNet.Common.ItemTemplates stabilizes, but the source build intermediate package does not. So in the rtm build we can't find the intermediate template package, Is this the right fix?

Reply:
LGTM.